src/lib.rs

```diff
@@ -150,9 +167,40 @@ impl ApplicationBuilder {
         self.log_settings.everything = b; self
     }
 
+    pub fn runcommand (&mut self, command: Option<&str>) -> &mut Self {
+        self.command = match command {
+            Some(cmd) => match fs::canonicalize(cmd) {
+                Err(e) => {
+                    println!("Error while parsing --run option: {}", e);
```

Comment:
log message instead of print

Reply:
Does it make sense to log  `--run` specific errors?

Currently they look like this:

    $ torchbear --run /not-such-file
    Error while parsing --run option: No such file or directory (os error 2)
    $
    $ torchbear --run /
    Error while parsing --run option: Can't get parent directory
    $

Reply:
Absolutely.  Everything gets logged.  Some messages need to be displayed.  Error is the lowest threshold for what messages to be displayed.  And `log` will also nicely format messages of this type in red.

Reply:
I've converted the calls to `log::error!("--run: {}", e);`, but now there is no output at all.

Reply:
I'm pretty sure it's just `error!()`, not `log::error!`.  Check log/torchbear/*.log to see if that even did anything.

Reply:
Same thing: no output.
I see, that logger is initialized a few lines later:
https://github.com/foundpatterns/torchbear/blob/master/src/lib.rs#L180

But here is another thing: where should the CLI app store log files?